src/PaymentManager/Payment/PayPalSmartPaymentButton.php

```diff
@@ -185,11 +185,11 @@ public function startPayment(OrderAgentInterface $orderAgent, PriceInterface $pr
     /**
      * Handles response of payment provider and creates payment status object
      *
-     * @param StatusInterface $response
+     * @param mixed $response
      *
      * @return StatusInterface
      */
-    public function handleResponse(StatusInterface $response): StatusInterface
+    public function handleResponse(mixed $response): StatusInterface
```

Comment:
Looks like could be narrowed down to
```suggestion
    public function handleResponse(StatusInterface|array $response): StatusInterface
```

See: https://github.com/pimcore/pimcore/blob/2b964d491f9d91cc7794357358861eaeed611919/bundles/EcommerceFrameworkBundle/src/CheckoutManager/V7/CommitOrderProcessor.php#L95-L106